faster_whisper/transcribe.py

```diff
@@ -482,7 +486,7 @@ def generate_with_fallback(
                 max_initial_timestamp_index=max_initial_timestamp_index,
                 **kwargs,
             )[0]
-
+            no_speech_prob = result.no_speech_prob
```

Comment:
Since `no_speech_prob` is included in `result` which is already returned by the function, maybe you don't need to return `no_speech_prob` separately?